src/WebJobs.Extensions.Twilio/Config/TwilioSMSConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Twilio
+{
+    /// <summary>
+    /// Defines the configuration options for the Twilio binding.
+    /// </summary>
+    public class TwilioSmsConfiguration
+    {
+        internal const string AzureWebJobsTwilioAccountSidKeyName = "AzureWebJobsTwilioAccountSid";
+        internal const string AzureWebJobsTwilioAccountAuthTokenName = "AzureWebJobsTwilioAuthToken";
+
+        public TwilioSmsConfiguration()
+        {
+            var nameResolver = new DefaultNameResolver();
+            AccountSid = nameResolver.Resolve(AzureWebJobsTwilioAccountSidKeyName);
```

Comment:
The name resolver is used in the `IExtensionConfigProvider` implementation here:
https://github.com/Azure/azure-webjobs-sdk-extensions/pull/124/files#diff-39fe02541097467043407a687aaebdf3R72 
Again, following the pattern established by Sendgrig, but happy to move this type elsewhere if improves clarity.